packages/blazor-workspace/NimbleBlazor/README.md

```diff
@@ -94,6 +94,15 @@ As the `NimbleTable` is generic a client must supply its generic type in the mar
 
 For more information regarding the Blazor component lifecycle mechanisms (such as `OnAfterRenderAsync`), please consult the [Microsoft Blazor docs](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/lifecycle).
 
+### Supported Render Modes
+
+Nimble supports all of the [Blazor render modes](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0):
+- Interactive server-side rendering (interactive SSR) using Blazor Server: `RenderMode.InteractiveServer`
+- Interactive WebAssembly: Client-side rendering (CSR) using Blazor WebAssembly: `RenderMode.InteractiveWebAssembly`
+- Interactive Auto: Interactive SSR initially, then CSR on subsequent visits after the Blazor bundle is downloaded: `RenderMode.InteractiveAuto`
+- Static server-side rendering (static SSR): Default, when no render mode is specified
+  - ⚠️Warning: This render mode is not recommended for most use cases with Nimble. As the page is just rendered once server-side and then no state is maintained, you're unable to use event handlers or call methods on components. This also means that for components like the Nimble Table / Wafer Map, setting data can't be done easily (because the methods to do so will have no effect if called).
```

Comment:
"can't be done easily" Is there something to say that is more descriptive? "can't be done via the component methods"

Reply:
Updated